data/ka'het missions.txt

```diff
@@ -0,0 +1,220 @@
+			choice
+				`	"Which technology did you recover from the crashed Ka'het?"`
+					goto tech
+				`	"Wait, why did you call them 'shells' and not ships?"`
+					goto shells
+				`	"Have the Ka'het ever attacked the Remnant planets?"`
+			`	"Actually, we never met any Ka'het in the Ember Waste at all. The 'shells' we found had similar capabilities to the quantum keystone, meaning they could pass through the wormholes, so we don't know for sure why they remain there. Perhaps they aren't aware of their skills, or maybe they lost them over time."`
+			choice
+				`	"Wait, why did you call them 'shells' and not ships?"`
+					goto shells
+			label tech
+			`	"Back then we found large numbers of their EMP torpedoes, most of which in almost pristine conditions; their internal mechanisms were also simple enough we were able to reproduce them perfectly. We didn't find enough of the other systems in the 'shells' to do the same, but their tech still have been our main inspiration point for centuries."`
+			choice
+				`	"Wait, why did you call them 'shells' and not ships?"`
+			label shells
+			`	"You see, <first>," she says, "the Ka'het aren't normal spaceships. The debris we found were artificial exoskeletons, built around huge living beings; it's unlikely a species that big could have developed that sort of technology, so we presume someone must have created it for them. This could have happened in the Graveyard, since it's the only place where we know them to still live in, but we have such a limited data about that section of space that we can't say for sure." As she finishes talking, one of the people Taely's was talking with earlier comes closer and shows her something on a touchpad. "I have to go now, Captain," she tells you. "Good luck."`
+				decline
+	on decline
+		log "Asked Taely about the Graveyard and the Ka'het. She told me the previous contacts between them and the Remnant."
+		log "Factions" "Ka'het" `The Ka'het are aliens which live in an area south of the Ember Waste, called Graveyard, and attack everyone on sight. They are large lifeforms which can travel in space using artificial exoskeletons.`
+
+mission "First Contact: Ka'het: Quarg 1A"
+	name "Abandoned Ringworld"
+	description "Visit a Quarg station to ask them about the abandoned ring in the middle of Ka'het space."
+	landing
+	source "Querii Ka"
+	destination "Lagrange"
+	to offer
+		not "First Contact: Ka'het: Quarg 1B: offered"
+		not "First Contact: Ka'het: Quarg 1C: offered"
+		not "First Contact: Ka'het: Quarg 1D: offered"
+		has "First Contact: Quarg: offered"
+		not "First Contact: Hai: offered"
+		not "First Contact: Kuwaru Efreti: offered"
+	to complete
+		never
+	on offer
+		log "Discovered an uninhabited, complete Quarg ringworld in the middle of Ka'het space."
```

Comment:
```suggestion
		log "Discovered a complete but uninhabited Quarg ringworld in the middle of the Graveyard."
```
Perhaps expand this log to say something along the lines of "It seems strange that the Quarg would abandon a ringworld given how important they are to them."